### ISSUE TITLE
The hyperlink "Find out more about the Xamarin Inspector Experience Improvement Program" is not accessible using keyboard.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#286]
### Pre-requisites:
Launch the Visual Studio for Mac and Sign-in with Enterprise account.
Create projects:
1. Go to Visual Studio-> File-> New Solution. -> Multiplatform-> App-> Xamarin.Forms-> Blank Forms App.
2. Hit on Next button to create the project.
3. Hit on Debug(Play)/Build button to build/deploy the solution/project.
4. As per the choice/requirements, Change the Device platform like iOS/ Android and iPhone X iOS 11.3 etc.
5. Hit on Inspector icon button (i.e., in the top toolbar buttons of the VS) to inspect the project.
6. On hitting on inspector button, Xamarin Inspector is Launched. 

### Repro-Steps:
1. Follow the Pre-requisites to Launch the Xamarin Inspector from Visual Studio.
2. Navigate to Xamarin Inspector-> preferences->Hit On Enter key, which opens the General tab by default.
3. Navigate to Feedback tab-> Hit on Enter to open Feedback tab window.
4. Press on the Tab key to traverse all the interactive elements of this opened feedback tab.
5. Observe the keyboard focus next to Feedback tab item.
 
### Expected Behavior
Hyperlink should be accessible and it should be invoked using keyboard also.

### Actual Behavior
The below  hyperlink is not accessible using keyboard.
Hyperlink : 
"Find out more about the Xamarin Inspector Experience Improvement Program."

### User Impact:
The keyboard user won't be able to interact with this hyperlinks. 

![kb2_feedback-hyperlink_notaccessiblewithkb](https://user-images.githubusercontent.com/22956130/39980680-b4c151aa-576a-11e8-88e7-829f6ce60667.png)

 
### Environment
Visual Studio for Mac Enterprise [Version: 7.5 Preview (7.5 build 1222)]
Xamarin Inspector [Version: 1.4 (master/b3f92f9)]

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Inspector-MAC
Wipro
MAS2.1.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1830
